docs/checks/asset_preload.md

```diff
@@ -0,0 +1,54 @@
+# Prevent Manual Preloading of Assets (`AssetPreload`)
+
+Preloading can be a useful way of making sure that critical assets are downloaded by the browser as soon as possible for better rendering performance.
+
+Liquid provides multiple filters to [preload key resources][preload_key_resources] so they can be converted into `Link` headers automatically. This enables them to be discovered even faster, especially when combined with Early Hints that Shopify supports.
+
+## Examples
+
+The following examples contain code snippets that either fail or pass this check.
+
+### &#x2717; Fail
+
+```liquid
+<link href="style.css" rel="preload" as="style">
+<link href="image.png" rel="preload" as="image">
+<link href="script.js" rel="preload" as="script">
+```
+
+### &#x2713; Pass
+
+```liquid
+{{ 'style.css' | asset_url | stylesheet_tag: preload: true }}
+{{ images['image.png'] | image_url: width: 600 | image_tag: preload: true }}
+{{ 'script.js' | asset_url | preload_tag: as: 'script' }}
```

Comment:
Minor nit: I'd format it a little bit so that it would be slightly easier to read. 

```liquid
{{ 'script.js' | asset_url | preload_tag: as: 'script' }}
{{ 'style.css' | asset_url | stylesheet_tag: preload: true }}
{{ 
  product.featured_image 
    | image_url: width: 600 
    | image_tag: preload: true 
}}

Reply:
Fixed ✅